app/controllers/users_controller.rb

```diff
@@ -10,24 +10,18 @@ def show
         render :json => users
     end
 
-    def user_Todos
-        userId = User.find_by(id: current_user.id)
-        todos=Todo.where(id:userId)
-        render :json => todos
-    end
-
-    def update_Email
-        duplicated_Email=User.find_by(email: user_params[:email])
-        if duplicated_Email==nil
+    def update_email
+        duplicated_user = User.find_by(email: user_params[:email])
+        if duplicated_user.nil?
             user = User.find(params[:id])
             user.update(user_params)
             render :json => user
         else
-            render json: { status: 409, errors: ['既に登録されています'] }, status: 409
+            render json: { status: 409, error: ['既に登録されています'] }, status: 409
```

Comment:
ここはそのままでいいですよ！（配列なので）
→もし意図的に配列にしていないのであれば、そちらを直しましょう🖐

（ひとりごと）
ここの409はconflictっぽいことなんだけど微妙にニュアンスが違うんだよなあ・・
伝えるのむずい🤔

```suggestion
            render json: { status: 409, errors: ['既に登録されています'] }, status: 409
```